cowprotocol/cowswap
# 1809/invalid order on orders table

# Summary

Closes #1809 

Displaying invalid orders with label `failed`

Actually, also decided to refactor all invalid references to `failed` as it makes more sense internally, even though the backend calls this `invalid`

Activity details were already displaying failed
![Screenshot 2022-12-29 at 13 19 38](https://user-images.githubusercontent.com/43217/209961419-3b86d056-da90-4dc5-b28e-2aa773a72afb.png)
Now the orders table on limit orders view also
![Screenshot 2022-12-29 at 13 19 29](https://user-images.githubusercontent.com/43217/209961422-89f4eea2-e496-4339-8a89-9ccdc03ecbcb.png)


  # To Test

1. Place an ethflow order and make sure it fails by tweaking the gas limit
2. Once failed, check activity status
* Should be failed
3. Go to limit orders history tab
* It should show as failed

**Note:** Orders that were failed before will not be displayed because of the internal `invalid` to `failed` change. 
Not ideal but still not a problem since the original change where the `invalid` orders were introduced (https://github.com/cowprotocol/cowswap/commit/b53696f138d1b0ed0e0bb6ecf597cef7324f7671) hasn't reached prod yet.
